src/index.js

```diff
@@ -1,8 +1,28 @@
 /* eslint-disable import/no-unresolved */
 // eslint-disable-next-line
 import _ from 'lodash';
-import dataInsert from './modules/dataInput.js';
-import htmlInput from './modules/interaction.js';
+import populateScores from './modules/populateScores.js';
+import getScores from './modules/getGame.js';
+import createGame from './modules/createGame.js';
```

Comment:
- [OPTIONAL] Kindly specify the correct filename to avoid `Can't resolve file path` issue. The correct filename should be `createGames`. There is no need for `.js` file extension for the app would detect it itself. Thanks to es6 features.